src/main/markdown/steering.md

```diff
@@ -11,7 +11,8 @@ The steering committee consists of the following members (no particular order):
 
 * Ray Cromwell, [Google](https://www.google.com/about/)
 * Artur Signell, [Vaadin](http://www.vaadin.com)
-* Colin Alworth, [Sencha](http://www.sencha.com)
+* Justin, Hickman, [Sencha](http://www.sencha.com)
+* Colin Alworth, [RedCurrent](http://www.redcurrent.com)
```

Comment:
:+1: - I talked to Brandon and gave him the go-ahead to add this, with RedCurrent's name. We don't have *any* product shipped yet, so at least for now, as far as you know, all we do is GWT ;).

Eventually you would know better - we don't have any non-"internal only tool" app bits that are using GWT.